integration-test/1730-further-water-layer-name-dropping.py

```diff
@@ -211,13 +211,11 @@ def test_label_lake_athabasca_z5(self):
             }),
         )
 
-        # we should get a label placement point, and its zoom should have been
-        # adjusted. we should also have all the names at this point.
+        # we should also have all the names at this point.
         self.assert_has_feature(
             z, x, y, 'water', {
                 'kind': 'lake',
-                'label_placement': True,
-                'min_zoom': 5,
+                'min_zoom': 2,  # min_zoom changed at https://github.com/tilezen/vector-datasource/pull/2010/
```

Comment:
Only the `'label_placement': True` point should have names at this early zoom, and the `min_zoom` on that should probably be a different value besides 2 (like 7)?